docs/sources/panels-visualizations/visualizations/node-graph/index.md

```diff
@@ -112,11 +112,13 @@ Required fields:
-| ------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| title         | string        | Name of the node visible in just under the node.                                                                                                                                               |
-| subtitle      | string        | Additional, name, type or other identifier shown under the title.                                                                                                                              |
-| mainstat      | string/number | First stat shown inside the node itself. It can either be a string showing the value as is or a number. If it is a number, any unit associated with that field is also shown.                  |
-| secondarystat | string/number | Same as mainStat, but shown under it inside the node.                                                                                                                                          |
-| arc\_\_\*     | number        | Any field prefixed with `arc__` will be used to create the color circle around the node. All values in these fields should add up to 1. You can specify color using `config.color.fixedColor`. |
-| detail\_\_\*  | string/number | Any field prefixed with `detail__` will be shown in the header of context menu when clicked on the node. Use `config.displayName` for more human readable label.                               |
+| Field name    | Type          | Description                                                                                                                                                                                                                                                                                                                                                               |
+| ------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title         | string        | Name of the node visible in just under the node.                                                                                                                                                                                                                                                                                                                          |
```

Comment:
For me, the IDE does this automatically so that the table is aligned if the source is shown without line wraps, like
```
|----------|---------------|
|content|some more|
```
And here some of the descriptions are long so it aligns length to that.